### PR TITLE
feat: [List V2] Initial Floating Bar

### PR DESCRIPTION
**Pull request by cbachmeier**
First PR for the floating bottom bar for the new list page, hidden behind Listing V2 Feature Flag. Dynamically adjusts price and button status depending on user entered price. Overlay below the bar when items scroll underneath. Hides listing panel if flag is enabled until listing is started. Also deprecated some VE components.

Known issues to be resolved in follow up PRs:
- Bar width doesn't match grid width. This will be resolved when the grid layout is updated when we change over the Listing Bag to a centered modal.
- Bar overlaps into current Listing Bag. Will also be resolved when bag is moved to a centered modal
- Mobile. Currently unsupported and will get its own PR.
- Error states. Error states like listing below floor or above current list price will show a misaligned warning tooltip. This logic is being completely updated in its own PR.

Screen Shot 2023-01-11 at 11 41 43

Update:
- better handle very small numbers
Screen Shot 2023-01-11 at 12 36 25

### CODE REVIEW

File: src/nft/components/profile/list/ListPage.tsx [hunk 1]
```diff
@@ -64,7 +121,20 @@ export const ListPage = () => {
...
+  const isNftListV2 = useNftListV2Flag() === NftListV2Variant.Enabled
+
+  const totalEthListingValue = useMemo(() => getTotalEthValue(sellAssets), [sellAssets])
+  console.log(totalEthListingValue)
+  const anyListingsMissingPrice = useMemo(() => !!listings.find((listing) => !listing.price), [listings])
+  const [ethPriceInUSD, setEthPriceInUSD] = useState(0)
+
+  useEffect(() => {
+    fetchPrice().then((price) => {
+      setEthPriceInUSD(price || 0)
```
Comment (JackShort) [inline src/nft/components/profile/list/ListPage.tsx:135]: nit: `??` makes more sense than `||`

**Review by JackShort — APPROVED** [review]
a few nits

File: src/nft/components/bag/Bag.tsx [hunk 2]
```diff
@@ -349,7 +351,7 @@ const Bag = () => {
                 color="white"
                 textAlign="center"
                 onClick={() => {
-                  isMobile && toggleBag()
+                  ;(isMobile || isNftListV2) && toggleBag()
```
[the remark this answers is not in the document]
Comment (cbachmeier) [reply src/nft/components/bag/Bag.tsx:354]: this was auto added by the linter

File: src/nft/components/profile/list/ListPage.tsx [hunk 3]
```diff
@@ -64,7 +121,20 @@ export const ListPage = () => {
   const collectionsRequiringApproval = useNFTList((state) => state.collectionsRequiringApproval)
   const listingStatus = useNFTList((state) => state.listingStatus)
   const setListingStatus = useNFTList((state) => state.setListingStatus)
+  const sellAssets = useSellAsset((state) => state.sellAssets)
   const isMobile = useIsMobile()
+  const isNftListV2 = useNftListV2Flag() === NftListV2Variant.Enabled
+
+  const totalEthListingValue = useMemo(() => getTotalEthValue(sellAssets), [sellAssets])
+  console.log(totalEthListingValue)
```
[the remark this answers is not in the document]
Comment (cbachmeier) [reply src/nft/components/profile/list/ListPage.tsx:129]: ty

**Pull request merged**